### PR TITLE
Updates to readme and changelog before release

### PR DESCRIPTION
**Pull request by zeibura**
- references to Bitlang removed from readme
- note about extended sources folder added to readme
- client changelog updated with major updates
 
I think I need more info for the main changelog - we seem to list every PR here under the headers Features / Bug Fixes along with the PR numbers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.13.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.12.0...0.13.0)
+
+### Bug Fixes
+
+### Features
+
```
Comment (temanbrcom) [inline CHANGELOG.md:9]: Disable copybook analysis for extended documents #448 
Add support for Area A&B #437 
Add margin A check for FD and SD level indicators #440 
Performance improvement #425
Syntax coloring support #423

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.13.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.12.0...0.13.0)
+
+### Bug Fixes
+
```
Comment (temanbrcom) [inline CHANGELOG.md:7]: Dispose diagnostic messages when user closes the document #434

**Comment by zeibura** [comment]
Both changelogs now done, thanks @temanbrcom

**Comment by zeibura** [comment]
Client readme updated too

File: clients/cobol-lsp-vscode-extension/README.md [hunk 3]
```diff
@@ -107,6 +99,6 @@ The extension includes the following copybook support features:
 * Functionality to skip variable levels when called, reducing call time.
 * Find All References and Go To Definition functionalities.
     - **Find All References** identifies all occurrences of variables and paragraphs from copybooks in the code.
-    - **Go To Definition** enables you to right-click on any variable or paragraph to reveal a definition of the element. If the definition is in a copybook, the copybook opens.
+    - **Go To Definition** enables you to right-click on any variable or paragraph to reveal a definition of the element. If the definition is in a copybook, or the name of a copybook, the copybook opens.
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/README.md:102]: The "go to definition" for copybook names will be available only in the next release

Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/README.md]: Removed

**Pull request merged**